client/app/scripts/superdesk-desks/aggregate.js

```diff
@@ -66,6 +69,44 @@
             setupCards();
         };
 
+        /**
+         * Return true if the 'fileType' filter is selected
+         * param {string} fileType
+         * @return boolean
+         */
+        this.hasFileType = function(fileType) {
+            return this.selectedFileType === fileType;
+        };
+
+        /**
+         * Set the current 'fileType' filter
+         * param {string} fileType
+         */
+        this.setFileType = function(fileType) {
```

Comment:
I will refactor this as part of changing the settings at desk and workspace level.